### PR TITLE
Adds medical record descriptions for quirks without any

### PR DESCRIPTION
**Pull request by Vermidia**
## About The Pull Request

Adds medical record descriptions to the heterochromatic, signer, spacer, and voracious quirks.
## Why It's Good For The Game

More flavor text! And also means they show up on medical records or when using medical HUDs to view quirks.

Right now if you view someone with these quirks the line will be blank in the medical records, so it makes that less confusing.
## Changelog
:cl:
add: Heterochromatic, Signer, Spacer, and Voracious quirks are now properly accounted for in medical records.
/:cl:

### CODE REVIEW

**Review by Fikou — CHANGES_REQUESTED** [review]
If this should be on every quirk you should make it a unit test

**Comment by Vermidia** [comment]
Tried my hand and making the unit test.

It should account for the edge cases where medical records are only added after being applied (the prosthetic quirks do this)

File: code/modules/unit_tests/quirks.dm [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /// Ensure every quirk has a unique icon
 /datum/unit_test/quirk_icons
+// Make sure all quirks start with a description in medical records
+/datum/unit_test/quirk_initial_medical_records
```
Comment (Mothblocks) [inline code/modules/unit_tests/quirks.dm:4]: Put this above the `/Run()` for quirk_initial_medical_records

File: code/modules/unit_tests/quirks.dm [hunk 2]
```diff
@@ -19,3 +21,23 @@
...
+		var/datum/quirk/quirk = initial(quirk_type)
+
+		if(isnull(quirk.medical_record_text))
+			//Add quirk to a patient - so we can pass quirks that add a medical record after being assigned someone
+			patient.add_quirk(quirk_type)
+
+			quirk = patient.get_quirk(quirk_type)
+
+			if(isnull(quirk.medical_record_text))
+				TEST_FAIL("[quirk_type] has no medical record description!")
```
Comment (Mothblocks) [inline code/modules/unit_tests/quirks.dm:41]: TEST_ASSERT

File: code/modules/unit_tests/quirks.dm [hunk 3]
```diff
@@ -19,3 +21,23 @@
...
 		used_icons[icon] = quirk_type
+
+/datum/unit_test/quirk_initial_medical_records/Run()
+	var/mob/living/carbon/human/patient = allocate(/mob/living/carbon/human/consistent)
+
+	for(var/datum/quirk/quirk_type as anything in subtypesof(/datum/quirk))
+		if (initial(quirk_type.abstract_parent_type) == quirk_type)
+			continue
+
+		var/datum/quirk/quirk = initial(quirk_type)
```
Comment (Mothblocks) [inline code/modules/unit_tests/quirks.dm:32]: This line doesn't make any sense and is unnecessary. Use quirk_type directly

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Review by san7890 — CHANGES_REQUESTED** [review]
should be good to go after this one
